src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -2,16 +2,16 @@
 
 public class HanoiRhymer extends defaultCountingOutRhymer {
 
-int totalRejected = 0;
+	int totalRejected = 0;
 
 	public int reportRejected() {
 		return totalRejected;
 	}
 
 	public void countIn(int in) {
-	if (!callCheck() && in > peekaboo())
+		if (!callCheck() && in > peekaboo())
```

Comment:
Please add curly brackets to increse readability. This is not a necessary change but in projects it improves the readability of the code